### PR TITLE
[HOTFIX] update Travis.yml

### PR DESCRIPTION
**Pull request by scottinet**
# Description

Update travis.yml from #683

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -91,59 +81,77 @@ jobs:
...
+    # next.console.kuzzle.io
+    # ---------------------------------------
     - stage: build & deploy
-      name: "Build & Deploy to next.console.kuzzle.io"
+      name: 'Build & Deploy to next-console.kuzzle.io'
 
       if: type != pull_request && branch = 3-dev
 
+      env:
+        - DEPLOY_PATH=/kuzzle-v1/
```
Comment (alexandrebouthinon) [inline .travis.yml:110]:
```suggestion
        - DEPLOY_PATH=/kuzzle-v2/
```

File: .travis.yml [hunk 2]
```diff
@@ -66,23 +54,25 @@ jobs:
...
+    # console.kuzzle.io
+    # ---------------------------------------
     - stage: build & deploy
-      name: "Build & Deploy to console.kuzzle.io"
+      name: 'Build & Deploy to console.kuzzle.io'
 
       if: type != pull_request && branch = master
 
+      env:
+        - DEPLOY_PATH=/kuzzle-v1/
```
Comment (alexandrebouthinon) [inline .travis.yml:70]:
```suggestion
        - DEPLOY_PATH=/kuzzle-v2/
```

File: .travis.yml [hunk 3]
```diff
@@ -30,33 +31,20 @@ stages:
...
+      name: 'Linter'
       script:
         - npm install
         - npm run lint
 
-# ---------------------------------------
-# Unit Testing
-# ---------------------------------------
-    - stage: tests
-      name: "Unit Tests"
```
Comment (xbill82) [inline .travis.yml:47]: I think that unit tests are still present and relevant in 2-x, shouldn't we keep them?

File: .travis.yml [hunk 4]
```diff
@@ -66,23 +54,25 @@ jobs:
 
       script:
         - npm install
-        - CYPRESS_RETRIES=5 npm run e2e
+        - CYPRESS_RETRIES=5 npm run test:e2e
```
Comment (xbill82) [inline .travis.yml:57]:
```suggestion
        - CYPRESS_RETRIES=5 npm run e2e
```

Comment (xbill82) [reply .travis.yml:57]: BTW we should harmonize npm script names in all versions

File: .travis.yml (hunk 1 above)
Comment (scottinet) [reply .travis.yml:110]: err... admin console v2 is not for kuzzle v1?

Comment (xbill82) [reply .travis.yml:110]: yep, but this is for the job that deploys `3-dev`. Currently there's no job in this file that deploys 2-stable nor 2-dev.

**Pull request merged**